src/containers/AccountConnection.jsx

```diff
@@ -262,6 +263,12 @@ class AccountConnection extends Component {
               date={account && account.lastSync}
             /> }
 
+            {!success && account && account.auth && <KonnectorFolder
+              connector={connector}
+              account={account}
+              driveUrl={this.store.driveUrl}
```

Comment:
I still think that making this call in the App component is too early and not the right place. 
The closest asynchronous call seems to be [this one](https://github.com/cozy/cozy-collect/blob/master/src/containers/ConnectorManagement.jsx#L41) in `ConnectorManagement`. What about fetching the drive url right after ?